src/tests/test_dynamic_api/test_views_wfs.py

```diff
@@ -180,3 +124,72 @@ def test_wfs_non_default_dataset_not_exposed(
         )
         response = api_client.get(wfs_url)
         assert response.status_code == 404
+
+
+@pytest.mark.django_db
+def test_wfs_model_auth(api_client, geometry_authdataset_thing, fetch_auth_token, filled_router):
```

Comment:
I'd prefer if the tests remain grouped. When you revisit a test file it's nearly impossible to see where to insert the next test. Grouping in classes helps for this.